eva-accession-clustering/src/test/java/uk/ac/ebi/eva/accession/clustering/batch/io/clustering_writer/RemappedVariantsClusteringWriterTest.java

```diff
@@ -300,24 +328,176 @@ public void test_variant_not_clustered_rs_id_exist() throws Exception {
         assertEquals(3000000006L, clusteredVariants.get(0).getAccession().longValue());
 
         //assert submittedVariationOperationEntity
-        List<SubmittedVariantOperationEntity> submittedVariantOperationEntities = mongoTemplate.findAll(SubmittedVariantOperationEntity.class);
+        List<SubmittedVariantOperationEntity> submittedVariantOperationEntities = mongoTemplate.findAll(
+                SubmittedVariantOperationEntity.class);
         SubmittedVariantOperationEntity submittedVariantOperationEntity = submittedVariantOperationEntities.get(0);
         assertEquals(1, submittedVariantOperationEntities.size());
         assertEquals(5000000000L, submittedVariantOperationEntity.getAccession().longValue());
         assertEquals(EventType.UPDATED, submittedVariantOperationEntity.getEventType());
-        assertEquals("Clustering submitted variant 5000000000 with rs3000000006", submittedVariantOperationEntity.getReason());
+        assertEquals("Clustering submitted variant 5000000000 with rs3000000006",
+                     submittedVariantOperationEntity.getReason());
     }
 
-    private SubmittedVariantEntity getSubmittedVariantEntity(String assembly, String project, long start, Long rs, long ss, String remappedFrom) {
-        SubmittedVariant submittedClustered = new SubmittedVariant(assembly, TAXONOMY, project, CONTIG, start, T, A, rs);
+    /**
+     * This test is for testing a specific scenario in clustering where an already clustered variant is remapped but the
+     * remapped variant is at a different location or a different variant type.
+     *
+     * SubmittedVariantEntity                                                           ClusteredVariantEntity
```

Comment:
Description looks good but can you please organise it so it is easy to read?  You can put the ClusteredVariantEntity below and alight the column names with the data

Reply:
done